### PR TITLE
ENH: Implement frame plot

### PR DESCRIPTION
**Pull request by charlesdong1991**
This PR is mainly creating frame plot which is kind of base method, now, only line plot is added.

koalas plot:

Screen Shot 2019-08-26 at 9 22 39 AM

pandas plot:

Screen Shot 2019-08-26 at 1 17 28 PM

### CODE REVIEW

File: databricks/koalas/plot.py [hunk 1]
```diff
@@ -879,3 +885,155 @@ def pie(self, **kwds):
...
+        return plot_frame(self.data, x=x, y=y, kind=kind, ax=ax,
+                          subplots=subplots, sharex=sharex, sharey=sharey, layout=layout,
+                          figsize=figsize, use_index=use_index, title=title, grid=grid,
+                          legend=legend, style=style, logx=logx, logy=logy,
+                          loglog=loglog, xticks=xticks, yticks=yticks, xlim=xlim,
+                          ylim=ylim, rot=rot, fontsize=fontsize, colormap=colormap,
+                          table=table, yerr=yerr, xerr=xerr, secondary_y=secondary_y,
+                          sort_columns=sort_columns, **kwds)
+
+    def line(self, x=None, y=None, **kwargs):
```
Comment (HyukjinKwon) [inline databricks/koalas/plot.py]: Can we mimic pandas documenation (https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.plot.line.html#pandas.DataFrame.plot.line)? It's okay to don't include the images for now.

**Comment by charlesdong1991** [comment]
Screen Shot 2019-08-26 at 8 50 54 AM
Hi, @HyukjinKwon thanks for taking a look at the PR, and I changed the code to reflect your points in the review. And here I attach the output with the test example code.

**Comment by charlesdong1991** [comment]
thanks for your feedback, fair enough! I added it in the description. Once it's merged, then I will add other sub modules for frame plot. @HyukjinKwon

**Comment by HyukjinKwon** [comment]
Merged to master.

Thanks you for addressing my comments.

**Pull request merged**